src/js/game/Entities/Player.js

```diff
@@ -88,6 +95,9 @@ module.exports = class Player extends BaseEntity {
     const queueHasOne = this.queue.currentCommand && this.queue.getLength() === 0;
     const timeEllapsed = (+new Date() - this.lastMovement);
     const movementAlmostFinished = timeEllapsed > 300;
+    if (isMoving && timeEllapsed > 800) {
```

Comment:
what's the reasoning for this time limit? I'd love to see a comment explaining

Reply:
Fair, I'm not a fan of magic numbers either, but I was essentially using the same logic as the movementAlmostFinished check. Waiting for one command to finish before checking for others so that functionality doesn't get tangled.